README.md

```diff
@@ -2,7 +2,7 @@
 [![Marketplace](https://img.shields.io/jetbrains/plugin/v/9834-jfrog)](https://plugins.jetbrains.com/plugin/9834-jfrog)
 # JFrog Idea Plugin 
 
-JFrog IntelliJ IDEA plugin adds JFrog Xray scanning of Maven, Gradle, Go and Npm project dependencies to your IntelliJ IDEA.
+JFrog IntelliJ IDEA plugin adds JFrog Xray scanning of Maven, Gradle, Go, Npm, and PyPI project dependencies to your IntelliJ IDEA.
```

Comment:
Let's add the following to the description:
```
In addition to scanning your project dependencies locally, the plugin also allows tracking the status of your code, as it it is being built, tested and scanned in your CI server.
```
I also suggest adding the below line to the end of the above section, and removing the *Using JFrog IntelliJ IDEA plugin* section.
```
To learn how to use JFrog IntelliJ IDEA plugin, please visit the [JFrog IntelliJ IDEA Plugin User Guide](https://www.jfrog.com/confluence/display/XRAY/IDE+Integration).
```